assets/stylesheets/garnett.scss

```diff
@@ -0,0 +1,68 @@
+// ----- Main Stylesheets ----- //
+
+@import './style';
+
+
+// ----- Garnett Styles ----- //
+
+.container-global.container-global--giraffe {
```

Comment:
Generally in BEM you would avoid nesting and selectors with multiple classes. Though I imagine in this case you're constrained by having to override all the violations of BEM style in other parts of the CSS...